src/main/java/net/geforcemods/securitycraft/screen/EditSecretSignScreen.java

```diff
@@ -54,7 +62,13 @@ protected void init()
 		minecraft.keyboardListener.enableRepeatEvents(true);
 		addButton(new Button(width / 2 - 100, height / 4 + 120, 200, 20, ClientUtils.localize("gui.done"), button -> close()));
 		te.setEditable(false);
-		textInputUtil = new TextInputUtil(minecraft, () -> te.getText(editLine).getString(), s -> te.setText(editLine, new StringTextComponent(s)), 90);
+		textInputUtil = new TextInputUtil(() -> signText[editLine], (s) -> {

```

Comment:
```suggestion
		textInputUtil = new TextInputUtil(() -> signText[editLine], s -> {
``` Brackets around the parameters are unnecessary for lambdas with only one parameter.